summoner-cli/src/Summoner/Template/GitHub.hs

```diff
@@ -28,16 +28,22 @@ import Summoner.Settings (Settings (..))
 import Summoner.Text (tconcatMap)
 import Summoner.Tree (TreeFs (..))
 
+import qualified Data.Text as T
 
 gitHubFiles :: Settings -> [TreeFs]
 gitHubFiles Settings{..} =
-    [File ".gitignore" gitignore     | settingsGitHub]
+    [File ".gitignore" (gitignoreDefault <> gitignoreCustom) | settingsGitHub]
  ++ [File ".travis.yml" travisYml    | settingsTravis]
  ++ [File "appveyor.yml" appVeyorYml | settingsAppVeyor]
   where
-    -- create .gitignore template
-    gitignore :: Text
-    gitignore =
+
+    -- additional .gitignore
+    gitignoreCustom :: Text
+    gitignoreCustom = T.intercalate endLine settingsGitignore <> endLine
+
+    -- default .gitignore template
+    gitignoreDefault :: Text
+    gitignoreDefault =
```

Comment:
While you are on it, you probably know what is missing in our `.gitignore` file. I was trying to include everything related to Haskell projects, but it could be outdated. For example, probably you know something `nix` specific that we are missing and should include in here?